packages/atomic/src/components/common/generated-answer/generated-answer-common.tsx

```diff
@@ -193,15 +193,25 @@ export class GeneratedAnswerCommon {
       });
   }
 
-  private clickDislike = () => {
+  private clickDislike() {
     if (
       this.modalRef &&
       !this.props.getGeneratedAnswerState()?.feedbackSubmitted
     ) {
       this.modalRef.isOpen = true;
     }
     this.props.getGeneratedAnswer()?.dislike();
-  };
+  }
+
+  private onChangeAnswerStyle(answerStyle: GeneratedAnswerStyle) {
+    if (
+      this.props.getGeneratedAnswerState()?.responseFormat.answerStyle ===
+      answerStyle
+    ) {
+      return;
+    }
+    this.props.getGeneratedAnswer()?.rephrase({answerStyle});
```

Comment:
Id suggest reversing the condition so we don't need the implicit "else" clause.

```suggestion
    if (
      this.props.getGeneratedAnswerState()?.responseFormat.answerStyle !==
      answerStyle
    ) {
      this.props.getGeneratedAnswer()?.rephrase({answerStyle});
    }
```